scripts/common.py

```diff
@@ -820,3 +844,27 @@ def managed_dev_appserver(
     ]
     with managed_process(dev_appserver_args, shell=True, env=env) as proc:
         yield proc
+
+
+@contextlib.contextmanager
+def managed_firebase_auth_emulator():
+    """Returns a context manager to manage the Firebase auth emulator.
+
+    Yields:
+        psutil.Process. The Firebase emulator process.
+    """
+    # TODO(#11549): Move this to top of the file.
+    import contextlib2
```

Comment:
Why can't this go to the top of the file now? (What error do you get if you do that -- can you show the stacktrace?)

Reply:
Every single script fails with an `ImportError`. Here's a link of such failures: https://github.com/brianrodri/oppia/runs/1628177447

Reply:
Ah ok. Can you file an issue for this, or append it to https://github.com/oppia/oppia/issues/11549 if it's the same thing? (The latter is missing details of the logs and repro steps, which I think are important.)

Reply:
#11549 was intended for it, so I just appended a few extra details.